truss-chains/truss_chains/example_chainlet.py

```diff
@@ -1,31 +1,35 @@
+import random
+
+# for more on chains, check out https://truss.baseten.co/chains/intro
```

Comment:
Nit: Capitalize and end with punctuation.

Reply:
⚡